src/components/CertificateTemplates/tlds/sg/edu/rp/2018-P-CCA/index.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import MultiCertificateRenderer from "template-utils/MultiCertificateRenderer";
+import { approvedAddresses } from "../common";
+import RPTranscript from "./transcript";
+
+const templates = [
+  {
+    id: "transcript",
+    label: "Transcript",
+    template: RPTranscript
+  }
+];
+
+const RPAA2020PCCA = props => (
+  <MultiCertificateRenderer
+    templates={templates}
+    whitelist={approvedAddresses}
+    {...props}
+  />
+);
+
+RPAA2020PCCA.displayName = "2020-P-CCA Template";
```

Comment:
is this normal ? name is 2020 ,  file name uses 2018

Reply:
Sorry, it should be 2018 instead of 2020.